prep-javascript-variables/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+var fullName="Danally Ly"
+var numberOfPets="0"
```

Comment:
For numbers, we want to leave the quotes off. Otherwise, JavaScript thinks that we mean a string instead of a number.